core/src/components/alert/test/a11y/alert.e2e.ts

```diff
@@ -41,7 +41,12 @@ test.describe('alert: a11y', () => {
 
     await didPresent.next();
 
-    const results = await new AxeBuilder({ page }).analyze();
+    /**
+     * alert overlays the entire screen, so
+     * Axe will be unable to verify color contrast
+     * on elements under the backdrop.
+     */
+    const results = await new AxeBuilder({ page }).disableRules('color-contrast').analyze();
```

Comment:
Adding `position: relative` to the alert seems to have caused the color contrast test to start failing and/or exacerbated an existing issue, but I can't see any difference visually, so it seems to be an Axe detection issue rather than an actual contrast problem.

Reply:
Since there are layered elements with opacity (such as the backdrop), the `position: relative` with `z-index: 10` likely creates a new stacking context which can cause Axe to be unable to verify the color contract.